pytest_fixtures/broker.py

```diff
@@ -60,6 +60,20 @@ def satellite_latest(default_sat):
         yield sat
 
 
+@pytest.fixture(scope='module')
+def mod_satellite_latest(default_sat):
```

Comment:
Please keep naming of these fixtures consistent. 

`module_satellite_latest`

`module_destructive_sat`

Reply:
Updated